src/main/kotlin/com/frc2036/comp/GameController.kt

```diff
@@ -53,8 +56,8 @@ class GameController {
                             if (board.player1Key != null && board.player2Key != null) {
                                 // TODO: replace cascade with when
                                 when (tile) {
-                                    TileType.Player1 -> tournament.keys.indexOf(board.player1Key!!)
-                                    TileType.Player2 -> tournament.keys.indexOf(board.player2Key!!)
+                                    TileType.Player1 -> tournament.keys.indexOf(board.player1Key!!) + 1
+                                    TileType.Player2 -> tournament.keys.indexOf(board.player2Key!!) + 1
```

Comment:
The player with the first key has index 0 in `tournament.keys`, which looks the same to the frontend as an empty key.